City-of-Helsinki/drupal-hdbt
# UHF-6669: Add events public url value back to see all button.

# [UHF-6669](https://helsinkisolutionoffice.atlassian.net/browse/UHF-6669)


## What was done
* Add events public url value back to see all events button.

## How to install
* Test with SOTE instance
* Make sure your instance is up and running on latest dev branch.
  * `git pull origin dev`
  * `make fresh`
* Update the HDBT theme
  * `composer require drupal/hdbt:dev-UHF-6669-tarkenna-hakua-nappi`
* Run `make drush-cr`

## How to test


* [x] Go to page: https://helfi-sote.docker.so/fi/sosiaali-ja-terveyspalvelut/senioripalvelut/palvelukeskukset
* [x] Check that there is "Tarkenna hakua tapahtumat.hel.fi:ssä" button at the bottom of the event listing.
* [x] Check that the button links to Tapahtumat.hel.fi url that is same as defined in Events Paragraph Api URL field.

## Designers review



[UHF-6669]: https://helsinkisolutionoffice.atlassian.net/browse/UHF-6669?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
